### PR TITLE
Add vk.com pixel

### PR DESCRIPTION
**Pull request by Parkhomets**
Hi! I added a pixel that allows to monitor site visitors and to match them to the audience of the social network vk.com

### CODE REVIEW

File: src/technologies/v.json [hunk 1]
```diff
@@ -618,5 +618,15 @@
       "generator": "vibecommerce"
     },
     "website": "http://vibecommerce.com.br"
+  },
+ "vk.com": {
+    "cats": [
+      1
+    ],
+    "description": "vk.com is russian social network",
+    "html": "<img [^>]*src=\"[^\"]*vk\\.com/rtrg?p=VK-RTRG-*",
```
Comment (AliasIO) [inline src/technologies/v.json:627]: Please use `"dom": "img[src*='vk.com/rtrg?p=VK-RTRG-']"` instead of `html`.

File: src/technologies/v.json [hunk 2]
```diff
@@ -618,5 +618,15 @@
       "generator": "vibecommerce"
     },
     "website": "http://vibecommerce.com.br"
+  },
+ "vk.com": {
+    "cats": [
+      1
+    ],
+    "description": "vk.com is russian social network",
```
Comment (AliasIO) [inline src/technologies/v.json:626]: Please use 'VK is a Russian online social media and social networking service.'

File: src/technologies/v.json [hunk 3]
```diff
@@ -618,5 +618,15 @@
       "generator": "vibecommerce"
     },
     "website": "http://vibecommerce.com.br"
+  },
+ "vk.com": {
```
Comment (AliasIO) [inline src/technologies/v.json:622]: I think we should call this 'VK Pixel'.

**Comment by Parkhomets** [comment]
Ok, I corrected it

File: src/technologies/v.json [hunk 4]
```diff
@@ -618,5 +618,15 @@
       "generator": "vibecommerce"
     },
     "website": "http://vibecommerce.com.br"
+  },
+ "VK Pixel": {
+    "cats": [
+      1
```
Comment (AliasIO) [inline src/technologies/v.json:624]: Sorry, just noticed this. The category should be `10` ([Analytics](https://github.com/AliasIO/wappalyzer/blob/master/src/categories.json#L38)).

**Comment by Parkhomets** [comment]
Sorry, I didn't notice that. I corrected it.

**Comment by AliasIO** [comment]
Perfect, thanks!

**Pull request merged**